src/cart/crt_hlc.c

```diff
@@ -96,9 +109,35 @@ int crt_hlc_get_msg(uint64_t msg, uint64_t *hlc_out, uint64_t *offset)
 	return 0;
 }
 
-uint64_t crt_hlc2sec(uint64_t hlc)
+uint64_t crt_hlc2nsec(uint64_t hlc)
 {
-	return (hlc & ~CRT_HLC_MASK) / NSEC_PER_SEC;
+	return hlc / CRT_HLC_NSEC;
+}
+
+uint64_t crt_hlc_from_nsec(uint64_t nsec)
```

Comment:
why not crt_nsec2hlc ?

Reply:
OK. (crt_nsec2hlc doesn't start with "crt_hlc_". I generally avoid "2" in function names, and find x_to_y and x_from_y favorable.)